dictionaries/software-terms/src/README.md

```diff
@@ -2,43 +2,56 @@
 
 Please feel free to make suggestions, additions, and corrections.
 
-There is an ongoing attempt to have the terms in the correct file.
+## Adding Words
 
-## Software Terms
+Contributions are welcomed and encouraged, please read this whole file to figure out where to add words.
 
-This file should contain software terms and concepts. Rule of thumb: [software-terms.txt](./software-terms.txt) should contains
-terms and concepts taught in a software development course. General terms used while coding should be added to [coding-terms.txt](./coding-terms.txt).
-Due to how the file evolved, this file also contains tools, products, applications, and even some brands.
-If you see these, please move them into the correct file.
+### Dictionary structure
+
+### Computing Acronyms
 
-- [software-terms.txt](./software-terms.txt) - Concepts.
 - [computing-acronyms.txt](./computing-acronyms.txt) - Acronyms and abbreviations.
 
-## Coding Terms -- General words used while coding
+### Coding Terms -- General words used while coding
 
-[Coding Terms](./coding-terms.txt) is a good place for general words used while coding. These words are uncommon in general English speech and writing,
+- [coding-terms.txt](./coding-terms.txt) is a good place for general words used while coding. These words are uncommon in general English speech and writing,
 but encountered while writing code.
 
-- [coding-terms.txt](./coding-terms.txt)
+### Software Terms
+
+- [software-terms.txt](./software-terms.txt) - Concepts and terms taught in a software development course.
+
+> [!NOTE]
+> This file is unfortunately the result of years of contributions without clear rules.
+> It is known to contain words that should not be in this `software-terms` dictionary.
+> It contains words that should have been added to English or `en_shared` dictionaries.
```

Comment:
```suggestion
```

I don't think this is necessary to say. English is not always enabled while coding.